packages/hawtio/src/plugins/jmx/JmxContent.tsx

```diff
@@ -21,6 +21,8 @@ import { Chart } from '@hawtiosrc/plugins/shared/chart'
 import { Operations } from '@hawtiosrc/plugins/shared/operations'
 import { Attributes } from '@hawtiosrc/plugins/shared/attributes'
 import { JmxContentMBeans, MBeanNode } from '@hawtiosrc/plugins/shared'
+import { AttributeTable } from '../shared/attributes/AttributeTable'
+import { NodeNameTable } from '../shared/attributes/NodeNameTable'
```

Comment:
please avoid using relative paths for importing components outside of its own module. Use `@hawtiosrc` to refer to `shared` module components.